numserver.com.delegate-num-zone.json

```diff
@@ -3,12 +3,14 @@
    "providerName":"NUM Server",
    "serviceId":"delegate-num-zone",
    "serviceName":"NUM Zone Delegation",
+   "logoUrl": "http://numserver.com/images/logo-black.png",
    "version": 1,
    "shared": true,
    "sharedProviderName": true,
    "logoUrl":"",
```

Comment:
Noticed only now, `logoUrl` is doubled here. Not sure how it passes schema check... but still pls fix it ;)